### PR TITLE
feature: navbar in login view

### PR DESCRIPTION
**Pull request by gbrl-pnhr**
logic for the navbar on the login screen.

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,10 +1,16 @@
-<script lang="ts">
-  import { RouterLink, RouterView } from 'vue-router'
-  export default {}
-</script>
 <template>
-  <header>
+  <header v-if="hasSideBar">
```
Comment (VictoriaRSsousa) [inline src/App.vue]: encompass everything in one tag

File: src/App.vue [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+    </header>
+    <RouterView />
+  </main>
+</template>
+<script lang="ts">
+  import { RouterView } from 'vue-router'
+  export default {
+    computed: {
+      hasNavBar(): boolean {
+        return this.$route.meta.hasNavBar as boolean
```
Comment (VictoriaRSsousa) [inline src/App.vue]: add semicolon

Comment (VictoriaRSsousa) [reply src/App.vue]: s

Comment (VictoriaRSsousa) [inline src/App.vue]: .

**Review by VictoriaRSsousa — APPROVED** [review]

**Review by VictorManoel-Timbo — APPROVED** [review]

**Pull request merged**